src/api/procedures/types.ts

```diff
@@ -974,3 +974,9 @@ export type RegisterMetadataParams =
       value: string;
       details?: MetadataValueDetails;
     };
+
+export type ManageAllowedVenuesParams = {
+  action: 'allow' | 'disallow';
+  ticker: string;
+  venues: number[];
+};
```

Comment:
Since this type will be exported, and assuming the procedure to manage allowedVenues will sit under the Asset class, the `ManageAllowedVenueParams` should only have a list of venues like -
```
export type ManageAllowedVenueParams {
    venues: BigNumber[];
}
```
Notice that I made the type as `BigNumber` instead of number, that is because we have made sure in all of SDK to change number to BigNumber for uniformity